packages/workflow/tsconfig.build.json

```diff
@@ -2,7 +2,8 @@
 	"extends": ["./tsconfig.json", "../../tsconfig.build.json"],
 	"compilerOptions": {
 		"rootDir": "src",
-		"outDir": "dist"
+		"outDir": "dist",
+		"tsBuildInfoFile": "dist/build.tsbuildinfo"
```

Comment:
Why are we not specifying `tsBuildInfoFile` at `packages/workflow/tsconfig.json`?